frontend/templates/index.html

```diff
@@ -7,81 +7,86 @@
     <script src="https://kit.fontawesome.com/f24e197897.js" crossorigin="anonymous"></script>
 </head>
 <body>
-
 <header id="header" class="header">
-    <a href="/" ><img src="{{ url_for('static', filename='images/logo.png') }}" loading="lazy" alt="" class ="logo">
-    </a>
+    <a href="/"><img src="{{ url_for('static', filename='images/logo.png') }}"loading="lazy" alt="" class="logo"></a>
     <div class="navigation">
+        <a href="/" class="navigation-links navigation-selected">Home</a>
+        <div class="navigation-link-separator"><</div>
         <a href="/about" class="navigation-links" data-i18n="Whats-Riskify"></a>
     </div>
 </header>
+
 <div class="loader-wrapper" id="loader-wrapper">
     <span class="loader"><span class="loader-inner"></span></span>
     <div class="loader-text"data-i18n="Loading"></div>
 </div>
-<div class="center">
-    <h3 data-i18n="processes"></h3>
-    <table id="processTable">
-        <colgroup>
-            <col width="17.5%">
-            <col width="17.5%">
-            <col width="17.5%">
-            <col width="17.5%">
-            <col width="10%">
-            <col width="10%">
-            <col width="5%">
-            <col width="5%">
-        </colgroup>
-        <tr>
-            <th data-i18n="process"></th>
-            <th data-i18n="components"></th>
-            <th data-i18n="score"></th>
-            <th data-i18n="status"></th>
-            <th data-i18n="created"></th>
-            <th data-i18n="edited"></th>
-            <th></th>
-            <th></th>
-        </tr>
-    </table>
-</div>
 
-<div class="center">
+ <div class="dashboard-overview">
+
+     <div class="center table-container">
+         <h3 data-i18n="processes"></h3>
+         <table class="responsive-table" id="processTable">
+             <colgroup>
+                 <col width="17.5%">
+                 <col width="17.5%">
+                 <col width="17.5%">
+                 <col width="17.5%">
+                 <col width="10%">
+                 <col width="10%">
+                 <col width="5%">
+                 <col width="5%">
+             </colgroup>
```

Comment:
@twonkos brauchen wir das noch oder kann das raus durch deine col-x Klassen? Genauso in der Tabelle darunter

Reply:
@NimsaJ150 Ist entfernt!